shadows/httpclient/src/main/java/org/robolectric/shadows/httpclient/ShadowDefaultRequestDirector.java

```diff
@@ -134,6 +134,8 @@ protected void __constructor__(
   }
 
   /**
+   * Get the {@link HttpRequest} for the given index.
```

Comment:
What about "Get the sent {@link HttpRequest} for the given index"?

Reply:
Sure

Reply:
https://github.com/robolectric/robolectric/pull/7399/files#diff-462c07a5e5d173b71bc1b609db060848db01f654fec2781fe1f5283fd6960575R138